deploy.sh

```diff
@@ -156,11 +156,10 @@ esac
 
 DOCKER_COMPOSE_FILE="docker-compose.yml"
 
-export INDEXES="mc_search"
 export ESOPTS='{"timeout": 60, "max_retries": 3}' # 'timeout' parameter is deprecated
-export ELASTICSEARCH_INDEX_NAME_PREFIX="mc_search-*"
 export TERMFIELDS="article_title,text_content"
 export TERMAGGRS="top,significant,rare"
+export ELASTICSEARCH_INDEX_NAME_PREFIX
```

Comment:
Is the value set somewhere?

Reply:
Yes in the private config repo

Reply:
I thought we had agreed:
1. the index alias/prefix was fixed (it appears in multiple places, including templates)
2. the staging News Search API stack should access the production ES index (the only way to really try it out is against "real data")